script/cmake-build

```diff
@@ -107,6 +107,7 @@ OT_POSIX_SIM_COMMON_OPTIONS=(
     "-DOT_SNTP_CLIENT=ON"
     "-DOT_SRP_CLIENT=ON"
     "-DOT_SRP_SERVER=ON"
+    "-DOT_RCP_TX_WAIT_TIME_SECS=5"
```

Comment:
nit: maintain alphabetical order

Reply:
I did not realize earlier that they were alphabetized. Thanks for catching it.